src/ui/components/Events/Events.tsx

```diff
@@ -23,6 +23,7 @@ function CurrentTimeLine({ isActive }: { isActive: boolean }) {
 
 function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
   const { userSettings } = hooks.useGetUserSettings();
+  const [items, setItems] = useState([]);
```

Comment:
I don't understand why these were aded. Doesn't look like anything is referencing `items` or `setItems`?

Reply:
This PR is old, sorry about that.

(And yes, those lines were dumb)